web/src/test/resources/com/epam/rft/atsy/cucumber/Positions.feature

```diff
@@ -4,52 +4,182 @@ Feature: As the user
 
   Background:
     Given The user signed in
-    And the Beállítások menu point clicked
-    And the options screen appears
-    And the positions link is clicked
-    And the positions screen appears
-
-  Scenario: user can list existing positions
-    Then the list filled with positions appears on the page
-
-  Scenario Outline: user can save new position
-    When user enters "<title>" into the title
-    And the Mentés button clicked
-    Then the new position called <title> appears in the list of positions
+    And the user clicks on Beállítások-text in the header
+    And the user clicks on Pozíciók tab link
+
+  Scenario Outline: 1 UI check
+    Then the user should get epam logo in the header
+    And the user should get Beállítások link in the header
+    And the user should get Kilépés link in the header
+    And the user should get Beállítások title
+    And the user should get Pozíciók tab link
+    And the user should get Csatornák tab link
+    And the user should get Jelszó-változtatás tab link
+    #TODO And the user should get Pozíciók table with the positions
+    # Név, Műveletek, Edit icon and Delete icon test
+    And the user should get <title> subtitle
+    And the user should get position label
+    And the user should get position field
+    And the user should get Mentés button
+    And the user should get Mégsem button
     Examples:
-      | title      |
-      | Új pozíció |
+    | title               |
+    | Új pozíció felvétele|
```

Comment:
If there won't be more examples, then you should use the value inline, ie:

`And the user should get Új pozíció felvétele subtitle`